clients/typescript/src/satellite/process.ts

```diff
@@ -475,7 +475,7 @@ export class SatelliteProcess implements Satellite {
     }
   }
 
-  async _connectAndStartReplication(
+  _connectAndStartReplication(
     opts?: SatelliteReplicationOptions
   ): Promise<void | SatelliteError> {
```

Comment:
Just while we're here

```suggestion
  ): Promise<void> {
```